packages/tooling/fast-tooling/src/schemas/web-component.schema.ts

```diff
@@ -5,8 +5,7 @@
  */
 export default {
     $schema: "http://json-schema.org/schema#",
-    $id: "custom-element-definition",
-    id: "custom-element-definition",
+    $id: "https://fast.design/docs/custom-element-definition",
```

Comment:
Although this does not currently exist at this location, this must be a valid URI.